docs/ref/models/expressions.txt

```diff
@@ -127,6 +127,21 @@ Although ``reporter.stories_filed = F('stories_filed') + 1`` looks like a
 normal Python assignment of value to an instance attribute, in fact it's an SQL
 construct describing an operation on the database.
 
+For fields that are string-expressions
```

Comment:
This section mentions slicing, but not indexing single values from the string/array for which support is also being added.